pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -193,18 +191,28 @@ async def run(self):
                         justifications.append(justification)
 
             # Verify attached bugs
-            advisories = list(filter(lambda ad: ad > 0, impetus_advisories.values()))
-            if advisories:
-                if self.skip_attached_bug_check:
-                    logger.info("Skip checking attached bugs.")
+            if self.skip_attached_bug_check:
+                logger.info("Skip checking attached bugs.")
+            else:
+                major, minor = util.isolate_major_minor_in_group(self.group)
+                next_minor = f"{major}.{minor + 1}"
+                logger.info("Checking if %s is GA'd...", next_minor)
+                cincinnati = CincinnatiAPI()
+                graph_data = await cincinnati.get_graph(channel=f"fast-{next_minor}")
```

Comment:
don't see a reason for a variable here...?
```suggestion
                graph_data = await CincinnatiAPI().get_graph(channel=f"fast-{next_minor}")
```